client/coral-admin/src/containers/Header.js

```diff
@@ -0,0 +1,16 @@
+import {gql} from 'react-apollo';
+import withFragments from 'coral-framework/hocs/withFragments';
+import Header from '../components/ui/Header';
+
+export default withFragments({
+  root: gql`
+    fragment TalkAdmin_Header on RootQuery {
```

Comment:
Actually this could be turned into a query instead of fragment.

Reply:
I think this one is okay, because it expresses the needs from the Header component. And we will have more needs in the Layout component that we will need to spread. No need to make an extra query out of this one.

Reply:
What do you expect will need more?

Reply:
We can take advantage of it. And preload stuff but now that I'm thinking that we only have that Header in the Layout, maybe it's not that needed as I thought.

Reply:
I can't think of any other cases, so that's why I think we can simplify it and just move the query directly to the header. But if you can think of some, go ahead :-)